Weather App/weather.py

```diff
@@ -46,77 +40,16 @@ def _get_weather_data(self, coordinate, start_date, end_date):
         r = requests.get(self.weather_API_url, params=weather_payload)
```

Comment:
Single letter naming. I will fix too by myself, np
https://peps.python.org/pep-0008/#names-to-avoid